finish/src/test/java/it/io/openliberty/guides/cors/TestCors.java

```diff
@@ -59,10 +59,12 @@ public void testPreflightCorsRequest() throws IOException {
     }
     // end::preflightTest[]
 
-    public void checkCorsResponse(HttpURLConnection connection, Map<String, String> expectedHeaders) throws IOException {
+    public void checkCorsResponse(HttpURLConnection connection, 
```

Comment:
These changes should also be done in the `start/` directory.

Reply:
Yup! I can update that one